### PR TITLE
Add Status Network Token to superchain token list

### PR DESCRIPTION
**Pull request by 0x-r4bbit**

### CODE REVIEW

File: data/SNT/data.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  "tokens": {
+    "ethereum": {
+      "address": "0x744d70fdbe2ba4cf95131626614a1763df805b9e"
+    },
+    "optimism": {
+      "address": "0x650AF3C15AF43dcB218406d30784416D64Cfb6B2"
+    },
+    "goerli": {
+      "address": "0x3D6AFAA395C31FCd391fE3D562E75fe9E8ec7E6a"
+    },
```
Comment (spacesailor24) [inline data/SNT/data.json:16]:
```suggestion
    }
```

**Comment by 0x-r4bbit** [comment]
The validation of this PR failed due to:

```
error: SNT on chain goerli token 0x3D6AFAA395C31FCd391fE3D562E75fe9E8ec7E6a has incorrect symbol
error: SNT on chain goerli token 0x3D6AFAA395C31FCd391fE3D562E75fe9E8ec7E6a has incorrect name
```

This is because on test nets, the token is called "Status Test Token" and "STT".
I've added `overrides` accordingly to account for that.

@spacesailor24 is this okay?

**Comment by spacesailor24** [comment]
Yes, that's exactly the update I was going to ask you for. Re-running tests now and will merge if they pass

Thank you!

**Comment by 0x-r4bbit** [comment]
@spacesailor24 Thank you!

Another question: how did you verify the token address for Optimism is indeed provided by Status?
What if this was just some malicious address that the bridge will then bridge to when SNT is locked up on L1?

**Comment by 0x-r4bbit** [comment]
@spacesailor24 @tremarkley I've added another goerli-optimism entry here.

I'd appreciate another review and possible merge - thanks!

**Comment by 0x-r4bbit** [comment]
Without intending to push too hard here:

Is there an ETA at when I can expect this to be merged/available in the standard bridge UI?

**Review by roninjin10 — APPROVED** [review]

**Pull request merged**